examples/natmod/stm32wb55_transparent_vcp/_stm32wb55_transparent.c

```diff
@@ -0,0 +1,639 @@
+#include <string.h>
+#include "py/dynruntime.h"
+#include "py/stream.h"
+#include "stm32wbxx_ll_utils.h"
+#include "stm32wbxx_ll_system.h"
+#include "ports/stm32/mpbthciport.h"
+#include "ports/stm32/rfcore.h"
+
+// Based on STM32CubeWB\Projects\P-NUCLEO-WB55.USBDongle\Applications\BLE\BLE_TransparentModeVCP
+
+// Don't enable this if stdio is used for transp comms.
+#define DEBUG_printf(...)  // mp_printf(&mp_plat_print, "rfcore_transp: " __VA_ARGS__)
+
+// TODO Check ble_init_params vs SHCI_C2_BLE_Init /
```

Comment:
Can this TODO be removed?

Reply:
This was a reminder to me to compare the rfcore / cpu2 init options in https://github.com/micropython/micropython/blob/203dae41fbaceeea9a04dc540b22de4ddd0d5a69/ports/stm32/rfcore.c#L568 to the comparable settings in the transparent mode project from ST found (mostly) here: https://github.com/STMicroelectronics/STM32CubeWB/blob/83aacadecbd5136ad3194f39e002ff50a5439ad9/Projects/P-NUCLEO-WB55.USBDongle/Applications/BLE/BLE_TransparentModeVCP/Core/Inc/app_conf.h#L107

While there are differences, most of them are things that are set to `0` in micropython but higher numbers in the examples. These are settings that relate to the Host portion of the stack though, which we're not using from ST. The main discrepancy is `MICROPY_HW_RFCORE_BLE_NUM_LINK (1)` compared to the examples generally being set to 2. That's a question for a different discussion however, and the ``MICROPY` settings can be overridden for particular projects if needed anyway.

Either way, the TODO is removed now.